modules/management/kubectl-apply/manifests/kueue-v0.9.1.yaml

```diff
@@ -12331,6 +12331,7 @@ spec:
       - args:
         - --config=/controller_manager_config.yaml
         - --zap-log-level=2
+        - --feature-gates=TopologyAwareScheduling=true
```

Comment:
Is TopologyAwareScheduling not enabled if we don't have the feature flag? Can you share a code pointer/documentation which explains this behavior?

Reply:
yes, by default TAS is disabled. https://kueue.sigs.k8s.io/docs/concepts/topology_aware_scheduling/#admin-facing-apis